graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/relay/PaginationHelper.java

```diff
@@ -152,10 +162,27 @@ private static <T> List<T> collectItems(Iterator<T> it, Arguments arguments, Cur
             if (arguments.before != null && cursorSupport.getCursor(value).equals(arguments.before)) {
                 break;
             }
+
+            //Adding a limit of 1000 to avoid OOM
```

Comment:
I think this comment is not up to date ?

Reply:
This is from a rebase, not originally in my PR...